src/main/webapp/organization.js

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+class Organization {
+  constructor(organization, isMaintainer = false) {
+    this.organization = organization;
+    this.isMaintainer = isMaintainer;
+    this.organizationElement = document.createElement("div");
+    this.createOrganization();
+  }
+
+  getOrganization() {
+    return this.organizationElement;
+  }
+
+  createOrganization() {
+    this.organizationElement.classList.add("organization");
+
+    const organizationNameElement = document.createElement('div');
+    organizationNameElement.classList.add("organization-name");
+    organizationNameElement.textContent = this.organization.name;
+
+    this.organizationElement.addEventListener('click', () => {
+      // TODO: GetOrganizationServlet to display more information about this servlet
+
+      const organizationPopupArea = document.getElementById("organization-popup-area");
```

Comment:
Instead, of this, can you consider either:
  fire a new event from the organizationElement; 
  { event = 'organization-selected' }  (you'll have to expose this.organizationElement as well)


  or pass an instance of an "OrganizationRenderer" to this class (which the search area can create and this class then can use). 

and then the search area can decide what to do / how to render the organization.

What you want to do is separate out the data from how it is rendered into the dom.

If you do the first thing, then the search area, after creating the organization can:

organization.organizationElement.attachEventListener('organization-selected', ()=> {
   const popup div = organization.createOrgPopup();
   setPopupTo(div);
});